CHANGES.rst

```diff
@@ -6,6 +6,13 @@
   constructor. It can also be controlled with the existing
   ``set_array_storage`` method of ``AsdfFile`` and the ``all_array_storage``
   argument to ``AsdfFile.write_to``. [#557]
+- Add new parameter `lazy_load` to ``AsdfFile.open``. It is ``True`` by
+  default and preserves the default behavior. ``False`` detaches the
+  loaded tree from the underlying file: all blocks are fully read and
+  numpy arrays are materialized. Thus it becomes safe to close the file
+  and continue using ``AsdfFile.tree``. However, `copy_arrays` parameter
+  is still effective and the active memory maps may still require the file
+  to stay open in case `copy_arrays` is ``False``. [#573]
```

Comment:
Sorry for another nit, but we should have double backticks around `lazy_load` and `copy_arrays` here so they render properly on github.

Reply:
Sure